src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,12 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <>
```

Comment:
no need to use fragment when you has common wrapper, if you had siblings without common parent, you would use fragment